### ISSUE TITLE
PyTorch Version Issue

### ISSUE THREAD

**Issue by JustinYuu** [facebookresearch/TimeSformer#52]
It seems that PyTorch has removed container_abcs from torch._six. after Pytorch 1.7.0. Therefore, when the version is >= 1.7.0, an import error will be thrown in timesformer/models/vit_utils.py, Line 14. I hope you could modify the version reminder from PyTorch >=1.6.0 to Pytorch==1.6.0, or changes the import code to 'import collections.abc as container_abcs' for the latter version

**Comment by gberta** [comment]
Thanks for pointing this out. Unfortunately, I can't modify the code anymore as I'm no longer affiliated with Facebook. Hopefully, this is not too big of an issue, and other users can figure out how to work around it.

**Comment by srijandas07** [comment]
@JustinYuu Yes, I have also encountered the same problem. I guess, downgrading Pytorch should be a solution.

**Comment by wanchichen** [comment]
I ran into the same issue in #47. Since @gberta won't be able to merge the PR, I'll leave [my fork](https://github.com/wanchichen/TimeSformer) with the fixes open for people to use.

**Comment by srijandas07** [comment]
@wanchichen its working now. Thanks
